tools/ctl/ctl.c

```diff
@@ -41,6 +41,13 @@ struct ctl_data {
 	bool no_abi;
 	/* component specific type, default 0 */
 	uint32_t type;
+	/* ipc version, 0: IPC3 or 1: IPC4, default 0 for IPC3 */
+	uint8_t ipc_version;
```

Comment:
Why not use "abi.major" directly. When we add IPC5, it's awkward to have this mapping (IPC5 would be 2).

Reply:
```ipc_version``` is an input from user. And ABI version determined by ipc_version. So without ipc_version being set, we don't know the ABI version.